client/src/rr/dnssec/keypair.rs

```diff
@@ -241,11 +242,12 @@ impl KeyPair {
     /// # Return
     ///
     /// the KEY record data
+    #[allow(deprecated)]
     pub fn to_sig0key(&self, algorithm: Algorithm) -> DnsSecResult<KEY> {
         self.to_public_bytes()
             .map(|bytes| {
                      KEY::new(Default::default(),
-                              Default::default(),
+                              KeyUsage::Zone,
```

Comment:
Due to the unclear situation, it's probably better to assume that people are still using KeyUsage::Zone keys for SIG(0) signatures. And KeyUsage::Host is also a possible value, in addition to the default of KeyUsage::Entity. So I guess `fn to_sig0key` should have a parameter `key_usage`?